src/lib/builders/tags-input/index.ts

```diff
@@ -22,15 +31,34 @@ export type CreateTagsInputArgs = {
 	// Whether the tags should be unique
 	unique?: boolean;
 	// What to do on blur
-	// blur?: 'nothing' | 'add' | 'clear';
+	blur?: 'nothing' | 'add' | 'clear';
+	// Whether the tag should be added on paste
+	addOnPaste?: boolean;
+	// Maximum number of tags (undefined || 0 or lower === no max)
+	maxTags?: number;
+	// Allowed tags
+	allowedTags?: string[];
+	// Denied tags
+	deniedTags?: string[];
```

Comment:
```suggestion
	// Allowed tags
	allowed?: string[];
	// Prohibited tags
	prohibited?: string[];
```

We can find another word if prohibited sounds too verbose.

Reply:
Disallowed?

Reply:
That could work!